flapjack/ext.py

```diff
@@ -198,12 +198,15 @@ def flatpak_builder(*args, check=None, distcheck=False):
         check_index, check_module = next(
             (ix, m) for ix, m in enumerate(manifest['modules'])
             if isinstance(m, dict) and m['name'] == check)
-        testcmd = 'make distcheck' if distcheck else 'make check'
+        testcmds = ['make distcheck'] if distcheck else ['make check']
         if check_module.get('buildsystem', None) == 'meson':
```

Comment:
Not part of these changes, but `dict.get(key)` already defaults to None, so no need to specify None as second parameter.